manifests/install.pp

```diff
@@ -17,6 +17,14 @@
     )
   }
 
+  if $pulp::enable_katello {
+    if $pulp::messaging_transport == 'rabbitmq' {
+      fail('pulp-katello plugin doesn\'t support rabbitmq')
```

Comment:
I don't think this is true? Given that the `pulp-katello` plugin is just adding a distributor type to Pulp.